diff --git a/config.def.h b/config.def.h
index 5ed8497550..35b3436117 100644
--- a/config.def.h
+++ b/config.def.h
@@ -8,15 +8,15 @@
 static int draw_input = 1;                  /* -noi option; if 0, the input will not be drawn by default */
 /* -fn option overrides fonts[0]; default X11 font or font set */
 static const char *fonts[] = {
-	"monospace:size=10"
+	"monospace:size=20"
 };
 static const char *prompt      = NULL;      /* -p  option; prompt to the left of input field */
 static const char *colors[SchemeLast][2] = {
 	/*     fg         bg       */
-	[SchemeNorm] = { "#bbbbbb", "#222222" },
-	[SchemeSel] = { "#eeeeee", "#005577" },
+	[SchemeNorm] = { "#c0b1c2", "#152126" },
+	[SchemeSel] = { "#e1e1e6", "#db498b" },
 	[SchemeOut] = { "#000000", "#00ffff" },
-	[SchemeCursor] = { "#222222", "#bbbbbb"},
+	[SchemeCursor] = { "#152126", "#c0b1c2" },
 };
 /* -l option; if nonzero, dmenu uses vertical list with given number of lines */
 static unsigned int lines      = 0;
