diff --git i/config.def.h w/config.def.h
index b832a90..29075f5 100644
--- i/config.def.h
+++ w/config.def.h
@@ -5,12 +5,12 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "Cozette:size=20:antialias=false:autohint=false";
 
 /* disable bold, italic and roman fonts globally */
-int disablebold = 0;
-int disableitalic = 0;
-int disableroman = 0;
+int disablebold = 1;
+int disableitalic = 1;
+int disableroman = 1;
 
 static int borderpx = 2;
 
@@ -153,7 +153,7 @@ static unsigned int cursorshape = 2;
  * Whether to use pixel geometry or cell geometry
  */
 
-static Geometry geometry = CellGeometry;
+static Geometry geometry = PixelGeometry;
 
 /*
  * Default columns and rows numbers
